Simple-Flu/simple-flu.ipynb

```diff
@@ -289,6 +260,202 @@
    "source": [
     "FRED is a poweful tool not just for understanding disease spread in the aggregate but also for exploring the individual behaviors that are contributing factors. As part of our baseline model, agents recorded where they were exposed to the respiratory disease."
```

Comment:
Since I noticed it:
```suggestion
    "FRED is a powerful tool not just for understanding disease spread in the aggregate but also for exploring the individual behaviors that are contributing factors. As part of our baseline model, agents recorded where they were exposed to the respiratory disease."
```